### DISCUSSION TITLE
Create session manually

### DISCUSSION THREAD

**Discussion by Malin88** [nextauthjs/next-auth#3699] [Help]
Hi, I have weird case...
Basically I have front end app where I use next-auth and social login. Additionally, I have an application with a user profile. When logging in through the front-end, it creates a special cookie, thanks to which the application with the profile knows that the user is logged in. Now I have to do it the other way. If the user logs in via the profile, this special cookie will be created, I want to create a session in next-auth so that the user is logged in. in this special cookie I keep the API token so I can get all user data. But how to create next-auth user sessions based on this? Anyone have any idea?

Thanks

**Comment by d1sna** [comment]
same problem! was it solved ?
